packages/contracts-core/package.json

```diff
@@ -24,7 +24,7 @@
     "lint:fix": "solhint --fix contracts/*.sol",
     "prettier": "prettier --write './contracts'",
     "prepublish": "yarn bootstrap && yarn build",
-    "test": "hardhat test"
+    "test": "FOUNDRY_PROFILE=core-ci forge snapshot --check && FOUNDRY_PROFILE=core-ci forge test"
```

Comment:
should this be running the `core-ci` profile even in local testing? should there be a separate `test:ci`

Reply:
That would mean that we need that script/profile in every package of the monorepo, correct? 

From what I saw, the CI called the same script in every package of the yarn project

Reply:
bigger question: should each package expose a `test:ci` that runs in CI? what behaviors do we want in CI but not in local testing?

Reply:
One that I know for sure is more runs for fuzzing